### PR TITLE
Rename cockpit-test-assets package to cockpit-tests

### PR DESCRIPTION
**Pull request by stefwalter**
We should start creating installable tests. This gets ready for that
with a well named package, that can include installable unit tests
and integration tests including (ideally) depending on the right
packages needed to run the tests.

### CODE REVIEW

**Comment by mvollmer** [comment]
Should we change the summary and description of the package as well?

File: tools/cockpit.spec [hunk 1]
```diff
@@ -576,17 +576,17 @@ cluster. Installed on the Kubernetes master. This package is not yet complete.
 # we only build test assets on rhel or if we're building from a specific commit
 %if %{defined wip} || 0%{?rhel}
 
-%package test-assets
+%package tests
 Summary: Additional stuff for testing Cockpit
```
Comment (mvollmer) [inline tools/cockpit.spec]: "Tests for Cockpit"?

File: tools/cockpit.spec [hunk 2]
```diff
@@ -576,17 +576,17 @@ cluster. Installed on the Kubernetes master. This package is not yet complete.
...
 Requires: %{name}-bridge >= %{required_base}
 Requires: %{name}-shell >= %{required_base}
 Requires: openssh-clients
 
-%description test-assets
+%description tests
 This package contains programs and other files for testing Cockpit, and
```
Comment (mvollmer) [inline tools/cockpit.spec]: "This package contains tests..."?

File: tools/cockpit.spec (hunk 1 above)
Comment (stefwalter) [inline tools/cockpit.spec]: Good catch. Fixed.

File: tools/cockpit.spec (hunk 2 above)
Comment (stefwalter) [inline tools/cockpit.spec]: Fixed.

**Pull request closed without merging**